### ISSUE TITLE
Odd behaviour in VCF and BED files - duplicate entries

### ISSUE THREAD

**Issue by adbeggs** [fritzsedlazeck/Spectre#9]
Hi Fritz,

I have run Spectre in single sample mode on a large cohort of Nanopore germline samples... I get the same output in the BED and VCF every time:

```
##Spectre_sample=D23_36899
#CHROM  POS     ID      REF     ALT     QUAL    FILTER  INFO    FORMAT  D23_36899
chr17   24225000        Spectre.DEL.0NCMBCW4    N       <DEL>   .       .       END=25683000;SVLEN=1458000;SVTYPE=DEL;CN=0      GT:HO:GQ        1/1:0.0:60
chr18   19362000        Spectre.DEL.I5UEYID3    N       <DEL>   .       .       END=20560000;SVLEN=1198000;SVTYPE=DEL;CN=0      GT:HO:GQ        1/1:0.0:60
chrX    254000  Spectre.DUP.LK59YCGJ    N       <DUP>   .       .       END=1925000;SVLEN=1671000;SVTYPE=DUP;CN=4       GT:HO:GQ        ./.:0.0:60
chr1_KI270709v1_random  2000    Spectre.DUP.DKZE5SK9    N       <DUP>   .       .       END=19000;SVLEN=17000;SVTYPE=DUP;CN=75  GT:HO:GQ        ./.:0.0:60
```

They're all being run in separate folders, within separate runs on the HPC so I can't explain this behaviour. When I look at the CN plots in the directory there are obvious chromosomal arm losses that are not being called in the BED file e.g.:

![plot-coverage-A21_32419-chr5](https://github.com/fritzsedlazeck/Spectre/assets/12495244/6a91d475-c94a-4c97-b9ec-c7c69637900d)

**Comment by philippesanio** [comment]
Hi @adbeggs, 

my apology for the late reply. We were working hard on updating Spectre to 0.2.0.

The duplicated calls as you call them are located within the problematic (e.g.  telomeric or centromeric) regions as far as I can see that when searching for the region on HG002. The new blacklist covers these regions and will resolve the issue you were describing.

The missing deletion you are show in the provided picture, might be due to a bug in v0.1.0. In particular, Spectre was discarding the last CNV of a chromosome, thus when writing the CNVs to BED or VCF files they where missing. This is fixed now. 

Please feel free to let me know if you find any other bugs in Spectre.  

Cheers
Philippe

**Comment by philippesanio** [comment]
Fixed in version 0.0.2 alpha
